yt_dlp/extractor/murrtube.py

```diff
@@ -0,0 +1,164 @@
+      name
+      __typename
+    }
+    __typename
+  }
+}"""})
+        meta = data['medium']
+
+        storage_url = 'https://storage.murrtube.net/murrtube/'
+        format_url = storage_url + meta['key']
+        thumbnail = storage_url + meta['thumbnailKey']
+
+        formats = []
+        if determine_ext(format_url) == 'm3u8':
+            formats.extend(self._extract_m3u8_formats(
+                format_url, video_id, 'mp4', entry_protocol='m3u8_native',
+                preference=1, fatal=False))
+        else:
+            formats.append({
+                'url': format_url,
+            })
+
+        return {
+            'id': video_id,
+            'title': meta['title'],
+            'description': meta['description'],
+            'formats': formats,
+            'thumbnail': thumbnail,
+            'duration': int(meta['duration']),
+            'uploader': meta['user']['name'],
+            'view_count': meta['viewsCount'],
+            'like_count': meta['likesCount'],
+            'comment_count': meta['commentsCount'],
+            'tags': meta['tagList'],
+            'age_limit': 18,
+        }
+
+
+class MurrtubeUserIE(MurrtubeIE):
+    IE_DESC = 'Murrtube user profile'
+    _VALID_URL = r'https?://murrtube\.net/(?P<id>[^/]+)$'
+    _TEST = {
+        'url': 'https://murrtube.net/stormy',
+        'info_dict': {
+            'id': 'stormy',
+            'age_limit': 18,
+        },
+        'playlist_mincount': 27,
+    }
+
+    def _real_extract(self, url):
+        username = self._match_id(url)
+        data = self._download_gql(username, {
+            "operationName": "User",
+            "variables": {
+                "id": username,
+            },
+            "query": """query User($id: ID!) {
+  user(id: $id) {
+    id
+    mediaCount
+    __typename
+  }
+}"""},
+            "Downloading user info")
+        user = data['user']
+
+        page_size = 10
+        num_pages = int(user['mediaCount']) // page_size + 1
+        entries = []
+
+        for page in range(num_pages):
+            data = self._download_gql(username, {
+                "operationName": "Media",
+                "variables": {
+                    "limit": page_size,
+                    "offset": page * page_size,
+                    "sort": "latest",
+                    "userId": user['id'],
+                },
+                "query": """query Media($q: String, $sort: String, $userId: ID, $offset: Int!, $limit: Int!) {
+  media(q: $q, sort: $sort, userId: $userId, offset: $offset, limit: $limit) {
+    id
+    __typename
+  }
+}"""},
+                "Downloading page {0}".format(page + 1))
+            media = data['media']
+
+            for entry in media:
+                entries.append(self.url_result(
+                    'murrtube:{0}'.format(entry['id']),
+                    MurrtubeIE.ie_key()))
+
+        playlist = self.playlist_result(entries, username)
+        playlist['age_limit'] = 18
+        return playlist
```

Comment:
```suggestion
        return self.playlist_result(entries, username)
```
age_limit is only needed for videos